tensorflow_lite_support/python/task/audio/audio_classifier.py

```diff
@@ -34,7 +34,8 @@
 class AudioClassifierOptions:
   """Options for the audio classifier task."""
   base_options: _BaseOptions
-  classification_options: _ClassificationOptions = _ClassificationOptions()
+  classification_options: _ClassificationOptions = \
+    _ClassificationOptions.create_from_pb2()
```

Comment:
Please use the Classification's dataclass default option. Protobuf actually don't allow you to set default values.

Reply:
@khanhlvg I've fixed them in relevant places.